syntax = "proto3";

package failure_detector;

service FailureDetector {
    rpc Ping (PingRequest) returns (AckResponse);
}

message PingRequest {
    string sender_id = 1;
}

message AckResponse {
    bool alive = 1;
}
